app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
```

Comment:
The `print_name` method of the `Animal` class could be a static method as it does not depend on any instance variables. This would make the code more efficient and easier to understand. Remember to use the `@staticmethod` decorator.